From 3faae53fc3f7b7c91970253a01f034b1e019ebcf Mon Sep 17 00:00:00 2001
From: Zhihuan He <huan.he@rock-chips.com>
Date: Thu, 12 Mar 2020 16:14:48 +0800
Subject: [PATCH] arm64: dts: rockchip: modify rk3368-808-evb vop dclk mode

modify vop-dclk-mode to 1 for rk3368-808-evb to fix
screen display glitter.

Signed-off-by: Zhihuan He <huan.he@rock-chips.com>
Change-Id: Ib13304b5e97626e76dafad91ba296f37b9b3159a
---
 arch/arm64/boot/dts/rockchip/rk3368-808-evb.dts | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk3368-808-evb.dts b/arch/arm64/boot/dts/rockchip/rk3368-808-evb.dts
index 4db345de03c9..e85962412104 100644
--- a/arch/arm64/boot/dts/rockchip/rk3368-808-evb.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3368-808-evb.dts
@@ -47,6 +47,11 @@
 	rockchip,pins = <1 11 RK_FUNC_1 &pcfg_pull_none_4ma>;
 };
 
+&dmc {
+	vop-dclk-mode = <1>;
+	status = "okay";
+};
+
 &isp_dvp_d2d9 {
 	rockchip,pins =
 			/* cif_data4 ... cif_data9 */
-- 
2.35.3

